### PR TITLE
Doc improvement in LKJCholesky distribution

### PR DESCRIPTION
**Pull request by felixdivo**
Better structure & formatting. Added more info to reference.

The change can be viewed here: https://docs-preview.pytorch.org/91091/distributions.html?highlight=lkjcholesky#torch.distributions.lkj_cholesky.LKJCholesky

### CODE REVIEW

File: torch/distributions/lkj_cholesky.py [hunk 1]
```diff
@@ -23,15 +23,17 @@ class LKJCholesky(Distribution):
     to make the probability of the correlation matrix :math:`M` generated from
     a Cholesky factor proportional to :math:`\det(M)^{\eta - 1}`. Because of that,
     when ``concentration == 1``, we have a uniform distribution over Cholesky
-    factors of correlation matrices. Note that this distribution samples the
+    factors of correlation matrices.::
```
Comment (kit1980) [inline torch/distributions/lkj_cholesky.py:26]: Remove full stop before colon?

Comment (felixdivo) [reply torch/distributions/lkj_cholesky.py]: Sure, makes sense. I though the double colon was just to produce the next box, but as it is rendered, of course, it needs to be removed.

```suggestion
    factors of correlation matrices::
```

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**